src/tiler/src/api.js

```diff
@@ -135,5 +172,16 @@ api.get(
     HTML_RESPONSE,
 )
 
+// 404 response
+// This works in production but breaks the dev server just by existing
+api.get(
```

Comment:
I think we'll need a more accommodating solution if we intend to keep this in.  Having to un/comment out this on develop doesn't seem long term sustainable.

Reply:
I've actually PR'd a fix for this on `claudia-local-api`, it just hasn't been bubbled up to Tilegarden yet.